.eslintrc.json

```diff
@@ -15,6 +15,8 @@
     "jsx-quotes": ["error", "prefer-double"],
     "compat/compat": ["error"],
 
+    "no-unused-vars": ["warn", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }],
```

Comment:
But it’s [part of Standard](https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json#L157) config, so here you are changing it to also not allow arguments.

@goto-bus-stop any preference here?

But changing from `error` to `warn` is a bigger deal, cause it will now allow commits with unused vars, which I’m not a fan of.

Reply:
Oh, I see you’ve commented out this line in the next commit.